### PR TITLE
GG-25993: Fix issue with trailing slashed in monitoring url.

### PR DESCRIPTION
**Pull request by smoldachev**

### CODE REVIEW

File: modules/management-console-agent/src/main/java/org/apache/ignite/agent/utils/AgentUtils.java [hunk 1]
```diff
@@ -64,7 +64,7 @@ private AgentUtils() {
      * @param clusterId Cluster ID.
      */
     public static String monitoringUri(String srvUri, UUID clusterId) {
-        return srvUri + "/clusters/" + clusterId + "/monitoring-dashboard";
+        return srvUri.replaceAll("/+$", "") + "/clusters/" + clusterId + "/monitoring-dashboard";
```
Comment (akuznetsov-gridgain) [inline modules/management-console-agent/src/main/java/org/apache/ignite/agent/utils/AgentUtils.java]: Could we have utility function that will be the same signature as `java.nio.file.Paths#get(java.lang.String, java.lang.String...)` ?
For example: `Utility.buildUri(String first, Object... more)` ? And hide all logic for `\` manipulation in utility function?
So usage will be: `buildUri(srvUri, "clusters", clusterId,  "monitoring-dashboard");`

Comment (nva) [reply modules/management-console-agent/src/main/java/org/apache/ignite/agent/utils/AgentUtils.java]: Try something like this or search in spring utils.
```
URI uri = URI.create(srvUri);

try {
    uri = new URI("http".equalsIgnoreCase(uri.getScheme()) ? "ws" : "wss",
        uri.getUserInfo(),
        uri.getHost(),
        uri.getPort(),
        "/clusters/" + clusterId + "/monitoring-dashboard",
        null,
        null
    );
}
catch (URISyntaxException x) {
    throw new IllegalArgumentException(x.getMessage(), x);
}
```

Comment (smoldachev) [reply modules/management-console-agent/src/main/java/org/apache/ignite/agent/utils/AgentUtils.java]: Replace on StringUtils#trimTrailingCharacter

File: modules/management-console-agent/src/main/java/org/apache/ignite/agent/utils/AgentUtils.java [hunk 2]
```diff
@@ -64,7 +65,7 @@ private AgentUtils() {
      * @param clusterId Cluster ID.
      */
     public static String monitoringUri(String srvUri, UUID clusterId) {
-        return srvUri + "/clusters/" + clusterId + "/monitoring-dashboard";
+        return URI.create(srvUri + "/clusters/" + clusterId + "/monitoring-dashboard").normalize().toString();
```
Comment (akuznetsov-gridgain) [inline modules/management-console-agent/src/main/java/org/apache/ignite/agent/utils/AgentUtils.java]: Lets create utility method for this?

Comment (smoldachev) [reply modules/management-console-agent/src/main/java/org/apache/ignite/agent/utils/AgentUtils.java]: Utility method for utility method? :) I think this method will only be used to monitoring URI.

**Pull request merged**